test/helpers/assets.ts

```diff
@@ -399,15 +399,12 @@ export async function registerForeignAsset(
 
   // Sanitize Xcm Location
   const xcmLoc = patchLocationV4recursively(xcmLocation);
-
+  const api = context.polkadotJs();
   const { result } = await context.createBlock(
-    context
-      .polkadotJs()
-      .tx.sudo.sudo(
-        context
-          .polkadotJs()
-          .tx.evmForeignAssets.createForeignAsset(assetId, xcmLoc, decimals, symbol, name)
-      )
+    api.tx.sudo.sudoAs(
```

Comment:
The sudo call can be removed here

Reply:
I enforced it because removing it is making all tests to fail because a balance discrepancy. I think it is harmless.